sdk/hanaonazure/azure-mgmt-hanaonazure/azure/mgmt/hanaonazure/version.py

```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.7.0"
+VERSION = "0.6.0"
```

Comment:
Why is this version getting downgraded?

Reply:
Automation issue, that's ok I'll fix it :)

Reply:
Thank you!